config/install/field.field.media.image.field_image_storage.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 1a7e8c9e-d18f-42a6-8e7c-632c9dde5aaf
+langcode: en
+status: true
+dependencies:
```

Comment:
is this done via the web UI? I hunted all over the place for a way to setup the dependencies but could not find anything